src/platform.ts

```diff
@@ -422,8 +426,9 @@ export class PlatformInformation {
                     if (archArray.length >= 2) {
                         let arch = archArray[1].trim();
 
-                        // Note: This string can be localized. So, we'll just check to see if it contains 32 or 64.
-                        if (arch.indexOf('64') >= 0) {
+                        if (arch.toUpperCase().indexOf('ARM')) {

```

Comment:
Not sure what all the different values could be but right now this would skip this if the arch was something like `arm64`

```suggestion
                        if (arch.toUpperCase().indexOf('ARM') >= 0) {

```

Reply:
good catch, I sent the PR too soon, I have some uncommitted changes locally and forgot to push.